Hymns/Display/DisplayHymnView.swift

```diff
@@ -17,7 +17,7 @@ struct DisplayHymnView: View {
                 ActivityIndicator().maxSize()
             } else {
                 VStack(spacing: 0) {
-                    DisplayHymnToolbar(viewModel: viewModel)
+                    DisplayHymnToolbar(viewModel: viewModel).padding(.top, 5)
```

Comment:
I think it already looks fine without the padding. Emily's bug was an old bug and I have since adding padding to the toolbar's components
![Simulator Screen Shot - iPhone 8 - 2020-06-15 at 19 08 08](https://user-images.githubusercontent.com/1427524/84723918-aa1a4100-af3b-11ea-9987-71581f4fb7ca.png)